manifests/secondarydns.yaml

```diff
@@ -0,0 +1,126 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: secondary
+---
+apiVersion: v1
+data:
+  USER_DOMAIN: secondary.io
+  DOMAIN_HOST_IP: 127.0.0.1
+  Corefile: |
+    .:53 {
+        auto {
+          directory /zones db\.(.*) {1}
+          reload 45s
+        }
+        reload
+        log
+    }
+kind: ConfigMap
+metadata:
+  name: secondary-dns
+  namespace: secondary
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: secondary
+rules:
+- apiGroups:
+  - kubevirt.io
+  resources:
+  - virtualmachineinstances
+  verbs:
+  - get
+  - list
+  - watch
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: secondary
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: secondary
+subjects:
+- kind: ServiceAccount
+  name: default
+  namespace: secondary
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    k8s-app: secondary-dns
+  name: secondary-dns
+  namespace: secondary
+spec:
+  progressDeadlineSeconds: 60
+  replicas: 1
+  selector:
+    matchLabels:
+      k8s-app: secondary-dns
+  template:
+    metadata:
+      labels:
+        k8s-app: secondary-dns
+    spec:
+      containers:
+      - args:
+        - -conf
+        - /etc/coredns/Corefile
+        image: k8s.gcr.io/coredns/coredns:v1.8.6
+        imagePullPolicy: IfNotPresent
+        name: secondary-dns
+        ports:
+        - containerPort: 53
+          name: dns
+          protocol: UDP
+        resources:
+          limits:
+            memory: 170Mi
+          requests:
+            cpu: 100m
+            memory: 70Mi
+        volumeMounts:
+        - name: config-volume
+          mountPath: /etc/coredns
+          readOnly: true
+        - name: secdns-zones
+          mountPath: /zones     
+          readOnly: true
+      - name: fedora
```

Comment:
Does it represent the `status-monitor`?

Reply:
yes, simply will be replaced with the new container
I used fedora so i could edit manually the file and see the plugin works and so on
(and because we dont have a container image for status monitor yet)

Imo we can merge this PR once the comments are done, and simply edit the file once we have the container image and such